customization-base/src/main/java/com/azure/autorest/customization/ClassCustomization.java

```diff
@@ -188,6 +204,27 @@ public PropertyCustomization getProperty(String propertyName) {
             propertyName);
     }
 
+    /**
+     * Gets the constant level customization for a constant in the class.
+     * <p>
+     * For instance properties use {@link #getProperty(String)}.
+     *
+     * @param constantName The constant name.
+     * @return The constant level customization.
+     */
+    public ConstantCustomization getConstant(String constantName) {
```

Comment:
Should we have a `removeConstant` method too?

Reply:
It can be added in the future when it's needed